src/home_work3/api/suggestionRoutes.js

```diff
@@ -0,0 +1,30 @@
+let express = require("express");
+let router = express.Router();
+const Joi = require("joi");
+const validator = require("express-joi-validation").createValidator({});
+
+let userModel = require("../models/UserModel");
+let UserService = require("../services/userService");
+
+let userService = new UserService(userModel);
+
+const queryParamSchema = Joi.object({
+  query: Joi.string().required()
+});
+
+router.get("/", validator.query(queryParamSchema), (req, res) => {
+  const query = req.query.query;
+  const limit = req.query.limit || 10;
```

Comment:
Consider storing the "10" magic number in constants.

Reply:
Resolved as per the comment